specification/schemas/PatchShipmentSurcharge.json

```diff
@@ -0,0 +1,41 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalShipmentSurcharge"
+    },
+    {
+      "required": [
+        "attributes"
+      ],
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "required": [
+            "amount",
+            "currency"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "name": {
+              "type": "string",
+              "example": "Re-labeling surcharge"
+            },
+            "description": {
+              "type": "string",
+              "example": "A carrier-specific (carrier_code) re-labeling surcharge"
+            },
+            "amount": {
+              "type": "integer",
+              "example": 523,
+              "description": "Surcharge amount in cents/pence."
+            },
+            "currency": {
```

Comment:
The `amount` and `currency` should be replaced with:
```
"price": {
  "$ref": "#/components/schemas/BasePrice"
},
```

Reply:
Gourav here. I thought about doing that at first. However, `price` is not an attribute name that would match the context of Surcharges, isn't it? It is not really a price we are talking about but exactly an amount that has currency.

Reply:
Agreed. We can think of another name to use for this combined property. In other occasions we have used `item_value`, `total_value` and `purchase_price`. What about `surcharge`?